server/src/main/java/io/spine/examples/chatspn/server/message/MessageAggregate.java

```diff
@@ -64,4 +68,35 @@ private void event(MessagePosted e) {
                  .setContent(e.getContent())
                  .setWhenPosted(e.getWhenPosted());
     }
+
+    /**
+     * Handles the command to edit a message content.
+     *
+     * @throws MessageContentCannotBeEdited
+     *         if the message not exists or is tried to be edited by non-sender
```

Comment:
" if the message does not exist, or editor is not an original sender of this message".

Please note: a) grammar fix, and b) comma, separating the logical parts of this sentence.